### DISCUSSION TITLE
While running SunbirdED-portal server, it's throwing error

### DISCUSSION THREAD

**Discussion by pradoshKumar** [Sunbird-Ed/Community#588] [Installation] [answered]
Hi I am setting up portal for my new laptop and  have followed the installation proces of Sunbird portal. But while running SunbirdED-portal server, it's throwing error. Attaching screenshot for your reference.

![Screenshot from 2023-07-23 21-09-46](https://github.com/Sunbird-Ed/Community/assets/33890313/27f16132-e1ca-4274-973c-bb3c17bca231)

**Comment by rajeevsathish** [comment, marked as answer]
@pradoshKumar as you can see that the error is saying that azure_storageService needs some env variables you need to pass the same from local to make the local work.
this is the values of the env variables to be updated by the user who is trying to use local to connect to a upstream.
cc: @vinukumar-vs @chitranshu-keshav
